xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetabledate.js

```diff
@@ -63,6 +63,21 @@ require(['jquery', 'jdateformatparser', 'daterangepicker', 'xwiki-events-bridge'
           updateInput(this, event, picker);
         });
 
+        
+        input.on('keypress', function (e) {
+          if(e.which !== 13) return;
+          var txt = input.val().trim();
+          var range = txt.split(' - ');
+          if (range.length == 2){
+            //range
+            hidden.val(moment(range[0], dateFormat).format('x')+'-'+moment(range[1], dateFormat).format('x'));
+          }
+          else{
```

Comment:
Else should be on the same line as the closing ``}`` in order to follow the code style.